README.md

```diff
@@ -10,10 +10,11 @@ The action is compatible with Windows, Linux, and OSX runners.  Windows self-hos
 * **solsta_client_secret:** Secret Key to authenticate usage of Solid State Networks console tools
 * **console_version:** Version of Solsta Console Tools to use
 * **scripts_version:** Version of Solsta Deploy Scripts to use
-* **create_type:**  Either product, repository, or environment to create/modify 
-* **target_product:**  Target product to create/modify (case-sensitive)
-* **target_environment:**  Target environment create/modify to(case-sensitive)
-* **target_repository:**  Target repository create/modify to (case-sensitive)
+* **action_type:**  Either create, or update
+* **object_type:**  Either product, repository, or environment to create/update 
```

Comment:
Don't the deploy scripts use "action" and "object"? Why use 'type`?

Reply:
Also the renaming between task/action should be addressed - if action is a better word then the deploy scripts should be changed.

Reply:
To me, "type" implies that there are finite choices of types, whereas "action" or "object" alone seem more open-ended.  I believed I was helping the user with those terms -- making the variable name part of the documentation, so to speak.

The scripts might have a good reason for keeping shorter arguments, since they have to be re-typed often, as opposed to the cut-pasting in a text editor that goes with github/gitlab.

That said, I do like "action" better than "task", as, again, "task" seems more open ended -- a task might be "spin the wheel", but an action is usually just the "spin" part.

Not sure there's any science here though, just my impressions about how words are used and understood. :)

Reply:
I suggest talking it through w/ Joel. Branching terminology is a real PITA when doing support.